spec/facility_spec.rb

```diff
@@ -2,25 +2,284 @@
 
 RSpec.describe Facility do
   before(:each) do
-    @facility = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+    @facility_1 = Facility.new(
+                  {name: 'Albany DMV Office', 
```

Comment:
So the more conventional way to break this out into multiple lines is
```ruby
@facility_1 = Facility.new({
   name: 'Albany DMV Office', address: '2242 Santiam Hwy 
   SE Albany OR 97321', phone: '541-967-2014' 
})
```
Do this for each of your facilities and your vehicles